test/yz_solrq_eqc.erl

```diff
@@ -141,15 +141,16 @@ prop_ok() ->
                 ),
 
                 ?PULSE(
-                    {SolrQ, Helper, IBrowseKeys, MeltsByIndex},
+                    {Helper, IBrowseKeys, MeltsByIndex},
                     begin
                         reset(), % restart the processes
                         unlink_kill(yz_solrq_worker_0001),
                         unlink_kill(yz_solrq_helper_0001),
                         unlink_kill(yz_solrq_eqc_fuse),
                         unlink_kill(yz_solrq_eqc_ibrowse),
-                        {ok, SolrQ} = yz_solrq_worker:start_link(yz_solrq_worker_0001),
-                        {ok, Helper} = yz_solrq_helper:start_link(yz_solrq_helper_0001),
+                        %%{ok, SolrQ} = yz_solrq_worker:start_link(yz_solrq_worker_0001),
```

Comment:
More importantly, we need to revisit this test altogether since we've changed the supervision structure enough that it doesn't pass any more. Forgot that this needs to be run with a `make clean && make pulse` otherwise the test doesn't run.